app/src/androidTest/java/com/example/sharingang/MainActivityTest.kt

```diff
@@ -22,8 +24,12 @@ class MainActivityTest {
     @get:Rule(order = 1)
     val activityRule = ActivityScenarioRule(MainActivity::class.java)
 
+    @get:Rule
+    val grantPermissionRule: GrantPermissionRule =
+        GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION)
+
     @Test
-    fun clickingOnButtonStartsMapActivity(){
+    fun clickingOnButtonStartsMapFragment(){
```

Comment:
```suggestion
    fun clickingOnButtonOpensMap() {
```
I think speaking of things in terms of how the User sees them instead of how the Android dev sees them makes sense.

Note: this file was likely not formatted either